metabase/metabase
# Remove color-related settings migration for 0.45

Epic https://github.com/metabase/metabase/issues/22826
Related to https://github.com/metabase/metabase/pull/23890. Please read its description first.

Removes migrations needed for 0.43 -> 0.44 upgrade. This PR should go in 0.45 and should not be backported.

How to test:
- Run Metabase OSS. Both Metabot and the lighthouse illustration should be visible on the home page.
- Run Metabase EE. Go to Admin -> Settings -> Appearance. Both Metabot and the lighthouse illustration settings should be available.
